agent/acs/client/acs_client.go

```diff
@@ -19,321 +19,51 @@
 package acsclient
 
 import (
-	"bytes"
-	"crypto/tls"
-	"encoding/json"
 	"errors"
-	"io"
-	"io/ioutil"
-	"net"
-	"net/http"
-	"net/url"
-	"reflect"
-	"strings"
-	"time"
 
-	"github.com/aws/amazon-ecs-agent/agent/ecs_client/authv4"
 	"github.com/aws/amazon-ecs-agent/agent/ecs_client/authv4/credentials"
 	"github.com/aws/amazon-ecs-agent/agent/logger"
-	"github.com/awslabs/aws-sdk-go/internal/protocol/json/jsonutil"
-	"github.com/gorilla/websocket"
+	"github.com/aws/amazon-ecs-agent/agent/websocket/client"
```

Comment:
Can you move files from `websocket/client` to `wsclient`.  I'm not a fan of having different package names from the directory name.